package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@bratislava/component-library",
-  "version": "1.0.0",
+  "version": "1.0.2",
```

The version should be changed only if there are changes to publish. This PR doesn't include any changes of that type. We should bump it to current version that we changed during our meet. 
```suggestion
  "version": "1.0.1",
```